deploy/toolchaincluster/member-sa.yaml

```
@@ -0,0 +1,109 @@
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: toolchaincluster-member
+  namespace: {{.Namespace}}
+rules:
+- apiGroups:
+  - toolchain.dev.openshift.com
+  resources:
+  - "idlers"
+  - "nstemplatesets"
+  - "memberoperatorconfigs"
+  - "memberstatuses"
+  - "toolchainclusters"
+  - "useraccounts"
```

Let's do the same as we did for host, just use the wildcard:
```suggestion
  - "*"
```